### PR TITLE
small emag tweak

### PR DESCRIPTION
**Pull request by cacogen**
makes emag work a bit better

### CODE REVIEW

**Comment by ATH1909** [comment]
emagception

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -92,6 +92,12 @@
 	log_combat(user, A, "attempted to emag")
 	A.emag_act(user)
 
+/obj/item/card/emag/emag_act(mob/user)
+	if(item_flags & EMAGGED)
+		return
+	item_flags |= EMAGGED
+	icon_state = "emag_alt"
```
Comment (PKPenguin321) [inline code/game/objects/items/cards_ids.dm]: image

**Comment by Tharcoonvagh** [comment]
So if you emag an emag it loses the shitty sprite

**Comment by PKPenguin321** [comment]
Closed because antag freeze btw

**Pull request closed without merging**